db/organizations/gabia.eno

```diff
@@ -0,0 +1,9 @@
+name: Gabia
+website_url: https://gabia.com/
+privacy_policy_url: https://policy.gabia.com/policy_collect
+privacy_contact: gprivacy@gabia.com
+country: KR
+description: "Gabia is providing IaaS type cloud service and SaaS type groupware which are required services for overall business IT department."
```

Comment:
Auto-translating from the Korea Wikipedia page results in this (if you feel the translation is correct, it could be a good summary):

Gabia is a cloud company located in Pangyo, Gyeonggi-do that provides a full range of services required for business IT, including IaaS-type cloud services, PaaS, and SaaS-type groupware solutions. Established on March 28, 1998.

Reply:
I do think the translated one is nice but I think the established date can be removed.